indico/modules/events/views.py

```diff
@@ -227,6 +228,8 @@ def __init__(self, rh, event_, **kwargs):
         self.event = event_
         kwargs['conf_layout_params'] = self._get_layout_params()
         kwargs.setdefault('page_title', self.sidemenu_title)
+        registered_users = [x.user for x in get_registered_event_persons(self.event)]
+        kwargs.setdefault('registered_users', registered_users)
```

Comment:
Shouldn't be here, this is called **a lot** and will result in poor performance for large events.